api/src/routes/edit.ts

```diff
@@ -126,6 +126,58 @@ function uploadFile(req, res, next) {
     });
 }
 edit.post("/object/:pid/datastream/:stream", requireToken, datastreamSanitizer, uploadFile);
+edit.post(
+    "/object/:pid/datastream/:stream/license",
+    requireToken,
+    bodyParser.json(),
+    datastreamSanitizer,
+    async function (req, res) {
+        const { pid, stream } = req.params;
+        const { licenseKey } = req.body;
+        try {
+            console.log("Uploading license", licenseKey);
```

Comment:
Do we need this console.log in the long term, or is this just for testing?

Reply:
Made change.